yudiencore/core.go

```diff
@@ -27,8 +27,18 @@ const (
 	type_map          = iota // map[string]interface{}
 )
 
+const ( // order matters for log levels
```

Comment:
This should be shared with the other file, perhaps in a struct.

Reply:
Seems like golang does allow const or enum to be put in a struct. It also seems that enum consts cannot be accessed from other packages. Following Geoff's format of enum consts for parts and type for now. Later we could refactor